public/locales/sv/translation.json

```diff
@@ -0,0 +1,26 @@
+{
+  "Morning": "Morgon",
+  "Afternoon": "Eftermiddag",
```

Comment:
Out of scope

Reply:
don't know how it jumped branches but I have deleted it now.